cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -680,6 +681,90 @@ def verify_name(source_usage_key, parent_usage_key, expected_name, display_name=
+        self.chapter_usage_key = self.response_usage_key(resp)
+
+        # create a sequential
+        resp = self.create_xblock(parent_usage_key=self.chapter_usage_key, display_name='seq1', category='sequential')
+        self.seq_usage_key = self.response_usage_key(resp)
+
+        resp = self.create_xblock(parent_usage_key=self.chapter_usage_key, display_name='seq2', category='sequential')
+        self.seq2_usage_key = self.response_usage_key(resp)
+
+        # create a vertical
+        resp = self.create_xblock(parent_usage_key=self.seq_usage_key, display_name='vertical1', category='vertical')
+        self.vert_usage_key = self.response_usage_key(resp)
+
+        resp = self.create_xblock(parent_usage_key=self.seq_usage_key, display_name='vertical2', category='vertical')
+        self.vert2_usage_key = self.response_usage_key(resp)
+
+        # create problem and an html component
+        resp = self.create_xblock(parent_usage_key=self.vert_usage_key, display_name='problem1', category='problem',
+                                  boilerplate='multiplechoice.yaml')
+        self.problem_usage_key = self.response_usage_key(resp)
+
+        resp = self.create_xblock(parent_usage_key=self.vert_usage_key, display_name='html1', category='html')
+        self.html_usage_key = self.response_usage_key(resp)
+
+    def assert_xblock_info(self, xblock, xblock_info):
+        """
+        Assert we have correct xblock info.
+        """
+        self.assertEqual(unicode(xblock.location), xblock_info['location'])
+        self.assertEqual(xblock.display_name, xblock_info['display_name'])
+        self.assertEqual(xblock.category, xblock_info['category'])
+
+    def assert_course_outline(self, course_outline):
+        """
+        Assert we have correct course outline data.
+        """
+        course = self.get_item_from_modulestore(self.usage_key)
+        self.assert_xblock_child_info(course, course_outline)
+
+    def assert_xblock_child_info(self, xblock, xblock_info):
+        """
+        Assert we have correct child xblock info.
+        """
+        xblock_children = xblock.get_children()
+        self.assertEqual(len(xblock_children), len(xblock_info.get('child_info', [])))
+        for child in xblock_children:
+            child_info = get_xblock_outline(child)
+            self.assert_xblock_info(child, child_info)
+            self.assert_xblock_child_info(child, child_info)
+
+    @ddt.data(1, 0)
+    def test_get_course_tree(self, fetch_course_tree):
+        """
+        Test that we get course tree.
+        """
+        for usage_key in (self.problem_usage_key, self.vert_usage_key, self.seq_usage_key):
+            xblock = self.get_item_from_modulestore(usage_key)
+            url = reverse('contentstore.views.xblock_handler', kwargs={'usage_key_string': unicode(usage_key)})
+            url = url + '?course_tree=' + str(fetch_course_tree) if fetch_course_tree else url
```

Comment:
Why we need 
`if fetch_course_tree else url` 
why not just 
`url = url + '?course_tree={}'.format(etch_course_tree)`?